examples/threading-timers/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "example-threading-timers"
+edition.workspace = true
+license.workspace = true
+publish = false
+
+[dependencies]
+ariel-os = { path = "../../src/ariel-os", features = ["time"] }
```

Comment:
"time" is still needed as Cargo feature, compared to "threading", which happens to have a laze module. Which is confusing. But not this PR's problem.